### ISSUE TITLE
Link to http://watch-learn.com/video-tutorials/tags/drupal-8/ is dead

### ISSUE THREAD

**Issue by brampauwelyn** [ivandoric/d8theming#3]
I get a server error while clicking on the link to `http://watch-learn.com/video-tutorials/tags/drupal-8/`

**Comment by ivandoric** [comment]
Yeah... I changed CMS of the site so there are some problems. Try this one: http://watch-learn.com/tags/tag:Drupal%208

**Comment by brampauwelyn** [comment]
Ok that one works 👍 

Thx
